tests/Stub/EntityStub.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Slavcodev Components
+ *
+ * @author Veaceslav Medvedev <slavcopost@gmail.com>
+ * @license http://www.opensource.org/licenses/bsd-license.php
+ */
+
+namespace Rebilly\Tests\Stub;
+
+use Rebilly\Rest\Entity;
+
+/**
+ * Class EntityStub.
+ *
+ * @author Veaceslav Medvedev <slavcopost@gmail.com>
```

Comment:
I will change before the merger. It will add a lot of changes, I do not want to complicate the diff